.travis.yml

```diff
@@ -1,12 +1,20 @@
 language: python
+services:
+  - redis-server
 python:
   - "3.5"
 install:
   - "pip install tox"
 script:
-  - "tox -v -e py35"
+   - "wget https://github.com/coreos/etcd/releases/download/v3.1.0/etcd-v3.1.0-linux-amd64.tar.gz"
+   - "tar xvzf etcd-v3.1.0-linux-amd64.tar.gz"
+   - "pushd etcd-v3.1.0-linux-amd64"
+   - "./etcd &"
+   - "popd"
```

Comment:
Just to mention, there's a backported `etcd` package for Ubuntu Trusty in this PPA:
https://launchpad.net/~cloud-green/+archive/ubuntu/trusty-backports

It's an older version, but I'm not aware of any minimum requirement for Commissaire.

Could maybe simplify `etcd` installation with this method:
https://docs.travis-ci.com/user/installing-dependencies/#Installing-Packages-from-a-custom-APT-repository